@navikt/core/react/src/layout/utilities/types.ts

```diff
@@ -27,3 +27,17 @@ type FixedResponsiveT<T> = {
 };
 
 export type ResponsiveProp<T> = T | FixedResponsiveT<T>;
+
+/* Darkside update */
+/* Border overlap */
+/* default */
+/* focus */
+/* strong */
+/* subtle */
+/* info */
+/* danger */
+/* warning */
+/* success */
+
+/* Bg overlap */
+/* bg-default */
```

Comment:
Are these comments supposed to be there?

Reply:
Removed them in the other PR to avoid merge-conflicts